src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -27,7 +27,7 @@ const NAME_CHARS_REGEX = /^[a-z0-9-.]*$/;
 const CLUSTER_NAME_START_END_REGEX = /^[a-z0-9](.*[a-z0-9])?$/;
 // NOTE: based on https://github.com/openshift/assisted-service/blob/master/internal/cluster/validations/validations.go#L32
 const CLUSTER_NAME_REGEX =
-  /^[.-a-z0-9]?(([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)*)[.-a-z0-9]?$/;
+  /^[-a-z0-9]?(([a-z0-9]([-a-z0-9]*[a-z0-9])?([a-z0-9]([-a-z0-9]*[a-z0-9])?)*)*)[-a-z0-9]?$/;
 const CLUSTER_NAME_NO_DOT_REGEX = /^[a-z0-9-]*$/;
```

Comment:
Isn't `CLUSTER_NAME_REGEX` what should be used?

Reply:
The interactive checker in OCM has more restrictions than what CIM allows now.
I'm guessing the regular expression could be simplified further but it was hard to understand the parts.

![checker](https://user-images.githubusercontent.com/829045/208624541-f30033ee-4241-4117-a160-ddb3969d2cde.png)

Reply:
I don't think `-test` is either a valid name or beginning of an URL.

I think both CIM and OCM should be unified - none of them use `dots`. And `dash (-)` is allowed in both cases. Max length is equal.

In both cases we end up with a k8s resource named by that string and later on building an URL out of it.

Reply:
@mareklibra I think you are right, in fact the `nameValidationSchema` uses the validation that the first and last characters are letters for both CIM and OCM.

PTAL, I set the same Regexp for both, one which doesn't allow the `dash` to be either at the beginning or end of the name.